extensions/docs/msal-node-extensions.md

```diff
@@ -63,35 +133,52 @@ const linuxPersistence = LibSecretPersistence.create(cachePath, serviceName, acc
 An unencrypted file persistence, which works across all platforms, is provided for convenience, although not recommended.
 
 ```js
-import { FilePersistence } from "@azure/msal-node-extensions";
+const { FilePersistence } = require("@azure/msal-node-extensions");
+
+const filePath = "path/to/cache/file.json";
+const filePersistence = await FilePersistence.create(filePath, loggerOptions);
+// Pass the persistence to msal config's cachePlugin
+msalConfig.cache.cachePlugin = new PersistenceCachePlugin(filePersistence);
+ // Initialize the electron authenticator
+authProvider = new AuthProvider(msalConfig); 
 
-const cachePath = "path/to/cache/file.json";
-const filePersistence = FilePersistence.create(cachePath, serviceName, accountName);
 ```
 
+> :https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/e7ea9fd970c035aaee9f8d3c3d3196334fd1c0de/extensions/msal-node-extensions/src/persistence/FilePersistence.ts#L18: If file or directory has not been created, `FilePersistence.create()` will create the file and any directories in the path recursively.
+
 ### Creating the cache plugin
 Create the PersistenceCachePlugin, by passing in the persistence object that was created in the previous step.
 
 ```js
-import { PersistenceCachePlugin } from "@azure/msal-node-extensions";
+const { PersistenceCachePlugin } = require("@azure/msal-node-extensions");
 
 const persistenceCachePlugin = new PersistenceCachePlugin(windowsPersistence); // or any of the other ones.
 ```
 
 To support concurrent access my multiple processess, the extensions use a file based lock. You can configure the retry number and retry delay for lock acquisition through CrossPlatformLockOptions.
 
 ```js
-import { PersistenceCachePlugin } from "@azure/msal-node-extensions";
+const {
+  PersistenceCreator
+  PersistenceCachePlugin,
+} = require("@azure/msal-node-extensions");
 
 const lockOptions = {
     retryNumber: 100,
     retryDelay: 50
 }
-const persistenceCachePlugin = new PersistenceCachePlugin(windowsPersistence, lockOptions); // or any of the other ones.
+
+const persistence = await PersistenceCreator.createPersistence(persistenceConfiguration);
+const persistenceCachePlugin = new PersistenceCachePlugin(persistence, lockOptions); // or any of the other ones
+// Pass the persistence to msal config's cachePlugin
+msalConfig.cache.cachePlugin = persistenceCachePlugin;
+// Initialize the electron authenticator
+authProvider = new AuthProvider(msalConfig);
+
 ```
 
 ### Setting the PersistenceCachePlugin on the MSAL Node PublicClientApplication configuration
```

Comment:
@bmahall let's remove this entire section (i.e. **Setting the PersistenceCachePlugin on the MSAL Node PublicClientApplication configuration**) and everything under it, since in the snippets above we already show how to create a PublicClientApplication instance with the persistenceCachePlugin. For more details, we can just direct people to check the samples

Reply:
I have updated the title , would like to keep this for the sake of completeness.  However, if absolutely redundant happy to have a separate PR to remove it. @derisen